input/fsh/profiles/FinnishSchedulingHealthcareService.fsh

```diff
@@ -0,0 +1,18 @@
+Profile: FinnishSchedulingHealthcareService
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the profile from base profiles.